langjo-frontend/src/Home.js

```diff
@@ -1,16 +1,16 @@
 import React from "react";
 import LevelTest from "./components/LevelTest/LevelTest";
-import BeginnerLevel from "./components/BeginnerLevel/BeginnerLevel"; 
-
+import BeginnerLevel from "./components/BeginnerLevel/BeginnerLevel";
 import BlocksCards from "./components/BlockCards/BlocksCards";
-
+import Hero from "./components/Hero/Hero";
 
 function Home() {
   return (
     <div>
       <LevelTest />
       <BeginnerLevel />
       <BlocksCards />
+      <Hero />
```

Comment:
<Hero /> should be in first one, put it on line 10 please.

Reply:
Done

Reply:
Done, thanks